homework/02LessonTwo.js

```diff
@@ -34,17 +40,34 @@ function equals(argument1, argument2) {
 // parameter1 is less than or equal to parameter2
 
 function lessThanOrEqualTo(parameter1, parameter2) {
+  if (parameter1 <= parameter2) {
+    return true;
+  } else {
+    return false;
+  }
+
 }
 
 // Problem 3:
 // write a function named add(number1, number2)
 // add will add two numbers and return the result
 
+function add(x, y) {
+  var x = 5;
+  var y = 6;
```

Comment:
I just started working on this to work ahead, so I'm not done, but would it involve the number() function? @zacacollier